Makefile

```diff
@@ -77,6 +77,7 @@ clean:
 	@echo "=== $(INTEGRATION) === [ clean ]: Removing binaries and temporary files..."
 	@rm -rfv bin package
 
+compile: CGO_ENABLED=0
```

Comment:
why using this syntax (I did not know) instaed of adding it directly in the command?

Reply:
"Make variables are not exported into the environment of processes make invokes... by default. However, you can use make's export to force them to do so. Change:"

Reply:
https://stackoverflow.com/a/23845391/3682217

Reply:
I used that syntax right after visiting the same StackOverflow thread. This way is set explicitly for the `compile` command (and we are using the same we use to set variables in `compile multiarch` above. WDYT?

Reply:
I also _tested_ it by temporally changing the echo comment to include the `CGO_ENABLED` value.

Reply:
ok, then I am not understanding the thread 😓

Reply:
> All examples have "export".
You are totally right, it might not work properly even if the echo showed the expected value.

> By the way I still prefer to have it explicit and not relying on make-file knowledge 
I agree, I'll try a different approach